rwkv/chat_with_bot.py

```diff
@@ -36,7 +38,8 @@
 assert prompt != '', 'Prompt must not be empty'
 
 print('Loading 20B tokenizer')
-tokenizer = tokenizers.Tokenizer.from_file('20B_tokenizer.json')
+tokenizer_path = Path(os.path.abspath(__file__)).parent / '20B_tokenizer.json'
```

Comment:
Just curious, how does it work?

Reply:
uses an absolute path to the `rwkv` folder instead of a relative path from the file. this way the script can be called from anywhere, it will always work. otherwise it just tries to find `20B_tokenizer.json` in the same folder the script was called